lib/cog/queries/bundles.ex

```diff
@@ -18,7 +19,7 @@ defmodule Cog.Queries.Bundles do
 
   def bundle_summaries() do
     from b in Bundle,
-    preload: [:namespace]
+    preload: [:namespace, :relay_groups]
```

Comment:
I'm wondering if both the bundle_details and bundle_summaries queries can go away since we now want to supply the commands list for both index and show. We can simply use `Repo.all` and `Repo.get` with preload of commands, namespace and relay_groups and call it a day.